hoomd/md/pair/aniso.py

```diff
@@ -248,3 +251,206 @@ def type_shapes(self):
+
+    For polytopes, computing interactions using a single contact point leads to
+    significant instabilities in the torques because the contact point can jump
+    from one end of a face to another in an arbitrarily small time interval. To
+    ameliorate this, the alj potential performs a local averaging over all the
+    features associated with the closest simplices on two polytopes. This
+    averaging can be turned off by setting the ``average_simplices`` argument
+    to ``False``.
+
+    .. py:attribute:: params
+
+        The ALJ potential parameters. The dictionary has the following keys:
+
+        * ``epsilon`` (`float`, **required**) - :math:`\varepsilon`
+            :math:`[energy]`
+        * ``sigma_i`` (`float`, **required**) - the insphere radius of the first
+            particle type.
+        * ``sigma_j`` (`float, **required**) - the insphere radius of the second
+            particle type.
+        * ``alpha`` (`int`, **required**) - Integer 0-3 indicating whether or
+            not to include the attractive component of the interaction (see
+            above for details).
+        * ``contact_ratio_i`` (`float`, **optional**) - the ratio of the contact
+            sphere radius of the first type with ``sigma_i``. Defaults to 0.15.
+        * ``contact_sigma_j`` (`float`, **optional**) - the ratio of the contact
+            sphere radius of the second type with ``sigma_j``. Defaults to 0.15.
+
+        Type: `TypeParameter` [`tuple` [``particle_types``, ``particle_types``],
+        `dict`]
+
+    .. py:attribute:: shape
+
+        The shape of a given type. The dictionary has the following keys per
+        type.
+
+        * ``vertices`` (`list` [`tuple` [`float`, `float`, `float`]],
+            **required**) - The
+            vertices of a convex polytope in 2 or 3 dimensions. The third
+            dimension in 2D is ignored.
+        * ``rounding` radii` (`tuple` [`float`, `float`, `float`] or `float`,
+            **required**) - The semimajor axes of a rounding ellipsoid. If a
+            single value is specified, the rounding ellipsoid is a sphere.
+        * ``faces`` (`list` [`list` [`int`]], **required**) - The faces of the
+            polyhedron specified as a list of list of integers.  The vertices
+            must be ordered (see :meth:`~.get_ordered_vertices` for more
+            information).
+
+    Specifying only ``rounding radii`` creates an ellipsoid, while specifying
+    only ``vertices`` creates a convex polytope (set ``vertices`` and ``faces``
+    to empty list to create the ellipsoid). To automate the computation of
+    faces, the convenience class method `get_ordered_vertices` can be used.
+    However, because merging of faces requires applying a numerical threshold to
+    find coplanar faces, in some cases `get_ordered_vertices` may result in not
+    all coplanar faces actually being merged. In such cases, users can
+    precompute the faces and provide them.
+
```

Comment:
Add formulas that explicitly document the potential energy computed, using diagrams if needed. We can check with @vyasr and the publication agreement to see if we are able to use figures from the paper or if we need to create new ones from scratch.

Reply:
I am not too familiar with the actual formulas. Honestly if @vyasr could add that section himself, or someone more familiar that would be great. I can eventually, but i don't know if that is the best use of my time.

Reply:
Yeah I'm happy to handle documenting this at some point in the next week or so.

Reply:
Well I obviously didn't get around to this in the promised time period, but taking a look again.

@joaander according to the author license [here](https://publishing.aip.org/wp-content/uploads/2019/10/AIPP-Author-License.pdf), we should be OK to reuse the figures from the paper. Specifically, the first clause under "Author Rights and Permitted Uses":
> Reprint portions of the Work (excerpts, figures, tables) in future works created by the Author, in keeping with professional publication ethics.
HOOMD documentation for this feature should count as a work created by an author (either of us), right?

Reply:
@b-butler I've updated the docstrings with the main formula and a bit more description of how it works. I also rendered the docs and they look fine to me except that the `params` and `shape` docs for the Type (the TypeParam) maybe need to indicate what the keys/values of the dict should be? I'm asking whether in a line like this one:
```
Type: `hoomd.data.typeparam.TypeParameter` [`tuple` [``particle_types``, ``particle_types``], `dict`]
```
the `dict` needs to have members in the same way that the `tuple`'s members are documented.

Reply:
Should this be resolved?